src/SFML/Graphics/Text.cpp

```diff
@@ -191,16 +191,50 @@ void Text::setStyle(std::uint32_t style)
 ////////////////////////////////////////////////////////////
 void Text::setFillColor(const Color& color)
 {
-    if (color != m_fillColor)
+    //In case we use the same color on all characters, no need to store
+    //the same color multiple times
```

Comment:
```suggestion
    // In case we use the same color on all characters, no need to store
    // the same color multiple times
```

Reply:
Check for other places where there is no space between `//` and the first word of the comment but I won't bother making comments for every single occurrence.